docs/config_search.md

```diff
@@ -187,9 +184,19 @@ Using the `--run-config-search-<min/max>...` CLI options you have the ability to
 
 Note: That by default quick search runs unbounded and ignores any default values for these settings
 
+## Ensemble Model Search
+
+_This mode has the following limitations:_
+
+- Can only be run in `quick` search mode
```

Comment:
Should we put a max submodel number here? At least as a recommendation?

Reply:
Not sure. I mention below that it should take less than one hour for a 4 submodel run, but if they want to run a larger submodel ensemble I don't see why we should restrict them.

Reply:
I think we should set expectations properly. Let's be upfront that it is only verified or expected to work properly on 4 or 5 and less